src/utils/childProcess.js

```diff
@@ -33,15 +33,20 @@ export function runCommand(cmd) {
     return new Promise((resolve, reject) => {
         const commands = cmd.split(SPACE);
         const [app, ...args] = commands;
-        const childProcess = spawn(app, args, { stdio: 'ignore' });
+        const childProcess = spawn(app, args);
+        let stdout = '';
+
+        childProcess.stdout.on('data', (data) => {
```

Comment:
If you want `data` to be a `String`, you would need to set encoding (i.e `childProcess.stdout.setEncoding('utf-8');`